### PR TITLE
Fix tab indent settings

### PR DESCRIPTION
**Pull request by Frederik-Baetens**

### CODE REVIEW

File: indent/crystal.vim [hunk 1]
```diff
@@ -632,4 +632,4 @@ endfunction
 let &cpo = s:cpo_save
 unlet s:cpo_save
 
-" vim:set sw=2 sts=2 ts=8 et:
```
Comment (bew) [inline indent/crystal.vim:635]: This line is a special line recognized by vim to handle automatically set some options for this file, it shouldn't be changed, see `:h modeline` in vim.

File: indent/crystal.vim [hunk 2]
```diff
@@ -632,4 +632,4 @@ endfunction
...
+set sw=2 sts=2 ts=8
```
Comment (bew) [inline indent/crystal.vim:635]: This is wrong, if you want to set the `shiftwitdh` option and some other, they should be in your own configs, not here!

**Comment by Frederik-Baetens** [comment]
Ah, sorry, but the indentation from the current plugin doesn't match the indentation CrystalFormat wants, so can I make a pr that matches that indentation?

**Comment by bew** [comment]
Checkout #33

**Comment by Frederik-Baetens** [comment]
Oh, thanks

**Pull request closed without merging**